data/phi0959/phi009/phi0959.phi009.perseus-lat2.xml

```diff
@@ -30,17 +30,22 @@
          <sourceDesc>
       	     <biblStruct>
       	        <monogr>
-      	           <author>P. Ovidius Naso</author>
-      	           <title>Ex Ponto</title>
-      	           <editor>
-      	      Arthur Leslie Wheeler
-      	    </editor>
+      	           <author xml:lang="lat">P. Ovidius Naso</author>
+      	           <title>Ovid</title>
+      	           <title type="sub">Tristia. Ponto</title>
+      	           <editor>Arthur Leslie Wheeler</editor>
       	           <imprint>
+      	              <pubPlace>London, UK</pubPlace>
       	              <pubPlace>Cambridge, MA</pubPlace>
+      	              <pubPlace>William Heinemann, Ltd.</pubPlace>
       	              <publisher>Harvard University Press</publisher>
-      	              <date>1939</date>
+      	              <date>1939 (printing)</date>
```

Comment:
we have also used `<date type="printing">` as opposed to putting it in parentheses

Reply:
That is probably better and I will do that moving forward and change it here.  That way you can search by attribute or add an attribute.